### PR TITLE
Removed the MSVC100 dependency requirement.

### PR DESCRIPTION
**Pull request by lfdversluis**
As we have recompiled them and adjusted our build script accordingly.

### CODE REVIEW

**Comment by lfdversluis** [comment]
@devos50 can you review? (it's only a deletion though)

File: doc/Building Tribler on Windows.md [hunk 1]
```diff
@@ -25,9 +25,8 @@ Inside the `build` folder, put the following items:
...
-5. `vc_redist_100.exe` (Microsoft Visual C++ 2010 Redistributable Package), which is available [here](https://www.microsoft.com/en-us/download/details.aspx?id=14632). In case you build 32 bit, get the x86 version [here](https://www.microsoft.com/en-us/download/details.aspx?id=5555). Again, don't forget to rename.
-6. `vc_redist_110.exe` (Visual C++ Redistributable for Visual Studio 2012), which is available [here](https://www.microsoft.com/en-us/download/details.aspx?id=30679). In case you build 32 bit, get the x86 version. Once more, don't forget to rename the file.
-7. `libsodium.dll` which can be downloaded from [libsodium.org](https://download.libsodium.org/libsodium/releases/) (as of writing version 1.0.8).
+5. `vc_redist_110.exe` (Visual C++ Redistributable for Visual Studio 2012), which is available [here](https://www.microsoft.com/en-us/download/details.aspx?id=30679). In case you build 32 bit, get the x86 version. Once more, don't forget to rename the file.
+6. `libsodium.dll` which can be downloaded from [libsodium.org](https://download.libsodium.org/libsodium/releases/) (as of writing version 1.0.8).
```
Comment (devos50) [inline doc/Building Tribler on Windows.md]: Libsodium 1.0.9 is already released ;)

Comment (lfdversluis) [inline doc/Building Tribler on Windows.md]: But we are not using it nor have tested it D:

**Comment by lfdversluis** [comment]
@whirm ready for merge

**Comment by lfdversluis** [comment]
retest this please

**Pull request merged**